src/test/regress/expected/multi_insert_select_conflict.out

```diff
@@ -300,23 +304,23 @@ ALTER TABLE target_table ADD CONSTRAINT fkey FOREIGN KEY (col_1) REFERENCES test
 BEGIN;
 	TRUNCATE test_ref_table CASCADE;
 NOTICE:  truncate cascades to table "target_table"
-	INSERT INTO 
-		target_table  
-	SELECT 
+	INSERT INTO
+		target_table
+	SELECT
 		col_2,
 		col_1
 	FROM source_table_1 ON CONFLICT (col_1) DO UPDATE SET col_2 = 55 RETURNING *;
-ERROR:  insert or update on table "target_table_1900000" violates foreign key constraint "fkey_1900000"
-DETAIL:  Key (col_1)=(1) is not present in table "test_ref_table_1900012".
-CONTEXT:  while executing command on localhost:57637
+ERROR:  insert or update on table "target_table_1900001" violates foreign key constraint "fkey_1900001"
+DETAIL:  Key (col_1)=(3) is not present in table "test_ref_table_1900012".
```

Comment:
Strange that this detail changes

Reply:
That's not related to this change, but how adaptive executor behaves. Depending from which shard the error comes, the error message changes (e.g., the execution order is not deterministic). 

We already ignore shard ids in the regression test output diffs (see src/test/regress/bin/normalize.sed)

 We have `multi_insert_select_conflict.out` test output changes not for this lines, but for the other lines above. However, while copying test output file  on to expected folder, we get such changes as well.

But, you're right, in general, it might be  nicer  not to include these changes in the diffs, but I cannot find an easy way of doing that.